content/operate/kubernetes/reference/yaml/multi-namespace.md

```diff
@@ -0,0 +1,67 @@
+---
+Title: Multi-namespace examples
+alwaysopen: false
+categories:
+- docs
+- operate
+- kubernetes
+description: YAML examples for deploying Redis Enterprise across multiple Kubernetes namespaces.
+linkTitle: Multi-namespace
+weight: 40
+---
+
+Multi-namespace deployment lets a single Redis Enterprise operator manage clusters and databases in different namespaces, providing better resource isolation and organization.
+
+Multi-namespace deployment enables:
+- Namespace isolation: Separate Redis Enterprise resources by team, environment, or application
+- Centralized management: Single operator manages multiple namespaces
+- Resource sharing: Efficient use of cluster resources across namespaces
+- Flexible RBAC: Fine-grained permissions per namespace
+
+This example shows:
+- Operator namespace: `redis-enterprise-operator` (where the operator runs)
+- Consumer namespaces: `app-production`, `app-staging` (where REC/REDB resources are created)
```

Comment:
Only REDBs are created in these separate namespaces; the REC lives in the operator's namespace.